Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -315,7 +315,10 @@ void Jit64::dcbz(UGeckoInstruction inst)
 	MOV(32, R(RSCRATCH), gpr.R(b));
 	if (a)
 		ADD(32, R(RSCRATCH), gpr.R(a));
-	AND(32, R(RSCRATCH), Imm32(~31));
+	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bWii)
+		AND(32, R(RSCRATCH), Imm32(~31));
+	else
+		AND(32, R(RSCRATCH), Imm32(Memory::RAM_MASK & ~31));
```

Comment:
I'll have to check the DBAT's later.

The root cause of this crash-to-desktop in JIT64 is the use of ADDR_MASK_MEM1 in the optimised version of DCBZ.  ADDR_MASK_MEM1 works for the PPC code being executed, but this game has sent an out of bounds address to DCBZ.  ADDR_MASK_MEM1 does not pick this up.  Dolphin then assumes that the address is within the physical address range and crashes.

JITIL and Interpreter do not crash here as they make no such assumptions.